adaptive/runner.py

```diff
@@ -54,6 +54,10 @@
 )
 
 
+def _key_by_value(dct, value):
+    return next((k for k, v in dct.items() if v == value), None)
```

Comment:
@akhmerov, do you think it would be better to raise `StopIteration` because now it's not possible to use `None` as a key.